templates/person/create-posting.html

```diff
@@ -7,15 +7,6 @@
     <h1>{% trans "Add a posting" %}</h1>
 </div>
 {% endblock %}
-{% block edit_links %}
-    <div class="col-sm-6">
```

Comment:
Since the `edit_links` block for this form is now identical to the corresponding block in `edit-posting.html`, and since this form inherits from `edit-posting.html`, we no longer need this block.